ui/helpers/utils/util.js

```diff
@@ -540,9 +538,27 @@ export function isNullish(value) {
 export function getSnapDerivationPathName(path, curve) {
   const pathMetadata = SNAPS_DERIVATION_PATHS.find(
     (derivationPath) =>
-      derivationPath.curve === curve && isEqual(derivationPath.path, path),
+      derivationPath.curve === curve &&
+      lodash.isEqual(derivationPath.path, path),
   );
 
   return pathMetadata?.name ?? null;
 }
 ///: END:ONLY_INCLUDE_IN
+
+/**
+ * The method escape RTL character in string
+ *
+ * @param {any} value
```

Comment:
maybe 
```suggestion
 * @param {*} value
```
ref: https://jsdoc.app/tags-param.html

Reply:
Good catch @digiwand , I will add this in a following PR.

Reply:
Done in this PR https://github.com/MetaMask/metamask-extension/pull/17537